rsocket/internal/SetupResumeAcceptor.cpp

```diff
@@ -197,61 +172,59 @@ void SetupResumeAcceptor::accept(
   auto processor = std::make_shared<OneFrameProcessor>(
       *this, transport, std::move(onSetup), std::move(onResume));
   connections_.insert(transport);
-  // transport can receive frames right away
+  // Transport can receive frames right away.
   transport->setFrameProcessor(std::move(processor));
 }
 
-std::shared_ptr<FrameSerializer>
-SetupResumeAcceptor::getOrAutodetectFrameSerializer(
-    const folly::IOBuf& firstFrame) {
-  if (defaultFrameSerializer_) {
-    return defaultFrameSerializer_;
+std::shared_ptr<FrameSerializer> SetupResumeAcceptor::frameSerializer(
+    const folly::IOBuf& frame) {
+  if (defaultSerializer_) {
+    return defaultSerializer_;
   }
 
-  auto serializer = FrameSerializer::createAutodetectedSerializer(firstFrame);
+  auto serializer = FrameSerializer::createAutodetectedSerializer(frame);
   if (!serializer) {
-    LOG(ERROR) << "unable to detect protocol version";
+    VLOG(2) << "Unable to detect protocol version";
     return nullptr;
   }
 
-  VLOG(2) << "detected protocol version" << serializer->protocolVersion();
+  VLOG(3) << "Detected protocol version " << serializer->protocolVersion();
   return std::move(serializer);
 }
 
-void SetupResumeAcceptor::closeAndRemoveConnection(
-    const yarpl::Reference<FrameTransport>& transport,
+void SetupResumeAcceptor::close(
```

Comment:
Too long, often clips past 80 columns.  I think reading `close(transport, exn)` is pretty unambiguous about closing `transport`.